django/db/backends/oracle/schema.py

```diff
@@ -248,3 +248,9 @@ def _collate_sql(self, collation, old_collation=None, table_name=None):
         if collation is None and old_collation is not None:
             collation = self._get_default_collation(table_name)
         return super()._collate_sql(collation, old_collation, table_name)
+
+    def _format_sql_for_default(self, wrapper_sql, default_sql, params):
+        # Oracle does not support parametrized database defaults
+        # so format here with quote_value instead.
```

Comment:
We have a feature flag for this `requires_literal_defaults`, so it's should be doable to get rid of this wrapper.